core/bubbles/textinput_bubble.ts

```diff
@@ -130,58 +123,6 @@ export class TextInputBubble extends Bubble {
     this.locationChangeListeners.push(listener);
   }
 
-  /** Creates and returns the editable text area for this bubble's editor. */
-  private static createTextArea(): HTMLTextAreaElement {
-    const textArea = document.createElementNS(
-      dom.HTML_NS,
-      'textarea',
-    ) as HTMLTextAreaElement;
-    textArea.className = 'blocklyTextarea blocklyText';
-    return textArea;
-  }
-
-  /** Creates and returns the UI container element for this bubble's editor. */
-  private createEditor(
-    container: SVGGElement,
-    textArea: HTMLTextAreaElement,
-  ): SVGForeignObjectElement {
-    const inputRoot = dom.createSvgElement(
-      Svg.FOREIGNOBJECT,
-      {
-        'x': Bubble.BORDER_WIDTH,
-        'y': Bubble.BORDER_WIDTH,
-      },
-      container,
-    );
-
-    const body = document.createElementNS(dom.HTML_NS, 'body');
-    body.setAttribute('xmlns', dom.HTML_NS);
-    body.className = 'blocklyMinimalBody';
-
-    textArea.setAttribute('dir', this.workspace.RTL ? 'RTL' : 'LTR');
-    body.appendChild(textArea);
-    inputRoot.appendChild(body);
-
-    this.bindTextAreaEvents(textArea);
-
-    return inputRoot;
-  }
-
-  /** Binds events to the text area element. */
-  private bindTextAreaEvents(textArea: HTMLTextAreaElement) {
-    // Don't zoom with mousewheel; let it scroll instead.
-    browserEvents.conditionalBind(textArea, 'wheel', this, (e: Event) => {
```

Comment:
Verify that we either don't need this or add it to the comment editor

Reply:
Good catch, factored it out to the comment editor.